core/service/src/engine/threshold/service/kms_impl.rs

```diff
@@ -578,6 +564,7 @@ where
         Role::indexed_from_one(config.my_id),
         session_maker,
     );
+    context_manager.load_mpc_context_from_disk().await?;
```

Comment:
I think we also need to add a similar implementation on the central KMS.  But maybe it is for a future issue?

Reply:
I believe we still need to add it (not in this PR of course). Since we need consistency between the threshold and central KMS. In particular we are still only running integration tests with the central KMS :/